manifests/base/deployment.yaml

```diff
@@ -224,3 +224,5 @@ spec:
                 secretKeyRef:
                   name: paypal-secret
                   key: webhook-id
+            - name: CACHE_TTL
+              value: 30000
```

Comment:
this needs to be quoted otherwise it will result in an error 
```suggestion
              value: "30000"
```

Reply:
can we make this 1 or 2 min? 30s is not a lot of time and for the things that we cache it's not a problem if they don't refresh for a while

Reply:
Changed it to 1 minute.